examples/avro-cli.py

```diff
@@ -36,19 +37,37 @@
 """)
 
 
-# User stores the deserialized user Avro record.
+"""
+    User stores the deserialized user Avro record.
+"""
+
+
 class User(object):
 
-    # Explicitly declare data members to prevent creation of dynamic properties
-    __slots__ = ["name", "favorite_number", "favorite_color"]
+    """
+        Slots are both more performant and memory efficient than the standard class __dict__.
+        This has the added benefit of adding stronger type safety by disabling dynamic attribute creation.
+        If the deserialized Avro object does not match the User class definition exactly an attribute error
```

Comment:
Is that really correct? The deserialized Avro object (dict) only needs to be a perfect subset of the User object, right? I.e., there could be additional members to User that are not reflected in the schema?